bolt/lib/Rewrite/DWARFRewriter.cpp

```diff
@@ -744,9 +754,12 @@ void DWARFRewriter::updateDebugInfo() {
           *DWODIEBuildersByCU.emplace_back(std::move(DWODIEBuilderPtr)).get();
       if (CU->getVersion() >= 5)
         StrOffstsWriter->finalizeSection(*CU, DIEBlder);
-      processSplitCU(*CU, **SplitCU, DIEBlder, *TempRangesSectionWriter,
-                     AddressWriter, DWOName, DwarfOutputPath, DWODIEBuilder);
+      ThreadPool.async([&, DwarfOutputPath, DWOName, CU, SplitCU] {
```

Comment:
Why are we capturing and passing these?

Reply:
They go out of scope and cause a seg fault.